server/models/Comment.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+// The unique ID for a Comment will be MongoDB's default _id, autogenerated.
+const commentSchema = mongoose.Schema({
+  parent: Number, // Optional, refers to the parent of the Comment, another Comment.
```

Comment:
These can be ID references to objects, in case we need to access the related object information in a view

`parent: { type: mongoose.Schema.types.ObjectId, ref: 'Post' }`